docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -525,6 +525,72 @@ public class MockGreetingServiceTest {
 ----
 <1> Since we configured `greetingService` as a mock, the `GreetingResource` which uses the `GreetingService` bean, we get the mocked response instead of the response of the regular `GreetingService` bean
 
+==== Using Spies instead of Mocks with `@InjectSpy`
+
+Building on the features provided by `InjectMock`, Quarkus also allows users to effortlessly take advantage of link:https://site.mockito.org/[Mockito] for spying on the beans supported by `QuarkusMock`.
+This functionality is available via the `@io.quarkus.test.junit.mockito.InjectSpy` annotation which is available in the `quarkus-junit5-mockito` dependency.
+
+Sometimes when testing you only need to verify that a certain logical path was taken, or you only need to override a single method and want the rest of the methods to be called normally.
```

Comment:
I think that we should be clear what "called normally" means and describe the lifecycle of the spied bean more thoroughly....

Reply:
So, we are linking to the Mockito site. Doing some cursory digging on the links there, I was able to find [This](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#13), where it has this block...

> Mockito *does not* delegate calls to the passed real instance, instead it actually creates a copy of it. So if you keep the real instance and interact with it, don't expect the spied to be aware of those interaction and their effect on real instance state. The corollary is that when an *unstubbed* method is called *on the spy* but *not on the real instance*, you won't see any effects on the real instance.

Based on that information, would you be okay with changing that last paragraph to...

> Sometimes when testing you only need to verify that a certain logical path was taken, or you only need to stub out a single method's response while still executing the rest of the methods on the Spied clone. Please see Mockito documentation for more details on Spy partial mocks.

This makes is clearer that we will not be executing the methods on the real instance and what happens in this test will be isolated to just the Spied clone. While still directing them to the full documentation that Mockito has already compiled on Spied objects.

Reply:
I like the idea... @geoand WDYT?

Reply:
I like it as well